source/ui/dataman/AM2DScanView.cpp

```diff
@@ -492,7 +492,52 @@ void AM2DScanView::mousePressEvent(QMouseEvent *e)
 
 void AM2DScanView::setPlotRange(double low, double high)
 {
-	spectrumView_->setPlotRange(low, high);
```

Comment:
Just curious, what's the difference between setPlotRange(...) and setEnergyRange(...) for the spectrum view? Or has this method just been renamed?